packages/manager/apps/catalog-revamp/src/public/translations/catalog-revamp/search/Messages_fr_FR.json

```diff
@@ -0,0 +1,4 @@
+{
+  "manager_catalog_search_placeholder": "Recherchez un produit",
+  "manager_catalog_search_button": "Afficher / Masquer les filtres"
```

Comment:
I think you either conjugate the 2 or leave the infinitive for both. From my point of view, I think it would be better to use the infinitive ("Rechercher un produit").

Reply:
Thank you for this, I will ask it to the product owner (I follow the design and specs)

Reply:
Done changed to `Rechercher un produit` (product owner agreed)